src/shrimps/views.py

```diff
@@ -21,3 +25,18 @@ def home(request):
     last_shrimp = Shrimp.objects.order_by('-birth_date')[0]
 
     return HttpResponse(f"""There are {num_shrimps} shrimps in our aquarium. Last shrimp born is {last_shrimp}""")
+
+
+def give_birth_to_shrimp(request):
+    """random shrimp generator: every time the page is loaded generates a new random shrimp"""
+
+    SHRIMP_NAME_LIST = ["Richard", "Gerard", "Niky", "Hubert", "Crevette", "Paul", "Eli", "Evi"]
+    # at first generate the random fields, then create a Shrimp model and populate it
+    baby_shrimp_weight = randint(10, 400)
+    baby_shrimp_size = randint(10, 100)
+    baby_shrimp_name = choice(SHRIMP_NAME_LIST)
+    baby_shrimp_color = choice(Shrimp.SHRIMP_COLOR_CHOICES)
```

Comment:
You are getting a tuple by calling the choice method on the list. Make sure you access it as a tuple and print only a string.